src/scss/components/btn.scss

```diff
@@ -1,3 +1,12 @@
+body{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-color: #333;
+}
+
+
```

Comment:
I'm sure that `body` styles shouldn't be kept in `components/btn.scss`. You should create a new file for it in `base/`